app/views/categories/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<section class="main-container">
+  <div class="headline">
+    <%= link_to categories_path do %>
+      <div class="arrow">&larr;</div>
+    <% end %>
+    <h2>TRANSACTIONS</h2>
+    <%= link_to new_category_cash_flow_path(@category) do %>
+      <div class="arrow">&rarr;</div>
+    <% end %>
+  </div>
+
+  <div class="transaction-container">
+    <div class="category-wrapper">
+      <div class="category-icon">
+        <img src="<%= @category.icon %>" class="icon">
+      </div>
+
+      <div class="category-details">
+        <%= link_to @category.name, class: "category-name" %>
+      </div>
+    </div>
+
+    <% @cash_flow.each do |cash_flow| %>
+      <div class="transaction-details">
+        <p class="name-amount"><%= cash_flow.name %>
+          <span>$<%= cash_flow.amount %></span>
+        </p>
+        <p class="date-time"><%= cash_flow.created_at.strftime('%Y-%b-%d') %>
+        at <%= cash_flow.created_at.strftime('%H:%M:%S') %></p>
+      </div>
+    <% end %>
+
+    <div class="total-transaction">TOTAL TRANSACTION
+      <span>$<%= @total_amount %></span>
+    </div>
+    
```

Comment:
**OPTIONAL** - You have done well so far, but in order to stick to the style and design of the page, please ensure that the total amount for the category is placed at the top of the page as it is seen in the original design template. Please note this 👍 👍 

Your project:
![wefwefwv](https://github.com/anita00001/budget-app/assets/73607512/61403bdd-48ee-49b8-b269-4a1b33b12c20)

Design template:
![Cwrgawrfgawr](https://github.com/anita00001/budget-app/assets/73607512/9ff63268-425d-4cb0-973d-cb534d542cf8)